requirements.txt

```diff
@@ -1,50 +1,11 @@
-appdirs==1.4.4
-attrs==20.1.0
-beautifulsoup4==4.9.1
-black==21.7b0
-certifi==2020.6.20
-chardet==3.0.4
-charset-normalizer==2.0.4
-click==7.1.2
-codecov==2.1.9
-coverage==5.2.1
-environs==8.0.0
-idna==2.10
-importlib-metadata==1.7.0
-inflect==4.1.0
-iniconfig==1.0.1
-joblib==0.16.0
-lxml==4.6.3
-marshmallow==3.7.1
-more-itertools==8.5.0
-mypy-extensions==0.4.3
-nltk>=3.6.4
-packaging==20.4
-pathspec==0.9.0
-pathtools==0.1.2
-pexels-api==1.0.1
-Pillow==8.3.2
-pluggy==0.13.1
-portalocker==2.0.0
-praw==7.5.0
-py==1.10.0
-pyparsing==2.4.7
-pytest==6.0.1
-pytest-cov==2.10.1
-python-dotenv==0.14.0
-python-pptx==0.6.18
-pyunsplash==1.0.0b9
-regex
-requests==2.24.0
-six==1.15.0
-soupsieve==2.0.1
-toml==0.10.1
-tomli==1.2.1
-tqdm==4.48.2
-tracery==0.1.1
-update-checker==0.18.0
-urllib3==1.25.10
-watchdog==0.10.3
-websocket-client==0.57.0
-XlsxWriter==1.3.3
-zipp==3.1.0
+python-pptx
```

Comment:
I wonder if adding the dependencies with a ">=" (e.g. `python-pptx>=0.6.18`) might be useful in case some of them have major updates.